IBM/carbon-addons-iot-react
# fix(Table): number data formatting fixes

**Summary**

- fix(Table): the span titles weren't being formatted correctly
- fix(TableCard): the numbers weren't being formatted correctly because I forgot to pass through the locale prop to the Table

**Change List (commits, features, bugs, etc)**

- docs(ValueCard): improve the component-level documentation there

**Acceptance Test (how to verify the PR)**

- Verify that if you mouseover the Table with localized values (in the Locale story) that the mouseover shows the same localization
- Verify that the Dashboard stories with different locales in their TableCard show their number formats differently
